### PR TITLE
Adding german language support.

### PR DESCRIPTION
**Pull request by poetaster**
hi! I've started to build up the dictionaries (one pulled from online, the second is the intersection with the Hunspell dictionary). I realized that the encoding is probably an issue. What encoding should I use? windows-1250 

I'm also not sure about the keypad layout, but I think this (all diacritical stuff on line one) is as clean as it gets. otherwise, it's german keyboard. I have more than the usual 'German' alphabet since a lot of French and Spanish kreep into wordlists. 

Let me know what you think! Thanks!

### CODE REVIEW

**Comment by monich** [comment]
Any single-byte encoding will do, as long as it's supported by Qt on Sailfish OS. The strings are compared by a straight memcmp and have to be sorted accordingly. I'll push the tool for sorting/preparing the words files over weekend, stay tuned.

File: data/de/language.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Deutsch",
+    "enconding": "UTF-8",
+    "alphabet": "abcdefghijklmnopqrstuvxyzäàáèéêöüß",
```
Comment (monich) [inline data/de/language.json:4]: I dropped the alphabet, it's redundant.

File: data/de/language.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Deutsch",
+    "enconding": "UTF-8",
```
Comment (monich) [inline data/de/language.json:3]: UTF-8 won't work, it has to be a single-byte encoding, e.g. windows-1252.

**Comment by poetaster** [comment]
Cool. I'll 
1. wait for your tool
2. remove the alphabet and
3. off to the races.

**Comment by poetaster** [comment]
Closing for rework.

**Comment by monich** [comment]
The tool is there, along with a [README](https://github.com/monich/harbour-wordle/blob/master/tools/README) file.

**Pull request closed without merging**